runtime/compiler/ilgen/Walker.cpp

```diff
@@ -3272,6 +3272,21 @@ TR_J9ByteCodeIlGenerator::genInvokeDynamic(int32_t callSiteIndex)
 
    if (comp()->getOption(TR_FullSpeedDebug) && !isPeekingMethod())
       comp()->failCompilation<J9::FSDHasInvokeHandle>("FSD_HAS_INVOKEHANDLE 0");
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+
+   TR::SymbolReference * targetMethodSymRef = symRefTab()->findOrCreateDynamicMethodSymbol(_methodSymbol, callSiteIndex);
+
+   loadFromSideTableForInvokeDynamic(callSiteIndex);
+
+   if (comp()->getOption(TR_TraceILGen))
+      printStack(comp(), _stack, "(Stack after load from callsite table)");
+
+   TR::Node* callNode = genInvokeDirect(targetMethodSymRef);
+
+   _invokeDynamicCalls->set(_bcIndex);
```

Comment:
The bit vector is used in the openj9 implementation to transform the call later. The new implementation doesn't need it.